roles/openshift_node/files/bootstrap.yml

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/ansible-playbook
+---
+- hosts: localhost
+  gather_facts: yes
+  vars:
+    origin_dns:
+      file: /etc/dnsmasq.d/origin-dns.conf
+      lines:
+      - regex: ^listen-address
+        state: present
+        line: "listen-address={{ ansible_default_ipv4.address }}"
```

Comment:
Do we really need to do this here?

Couldn't we just use 'bind-dynamic' option?

If we're binding to the public IP of the instance, it's probably okay to bind to all of them, unless I'm missing something.

We can just detect bootstrapping to use bind-dynamic in the normal role.  This might even been a good option for all nodes in general.

Reply:
I was told to not bind to localhost.  That's the reason for this.  @sdodson might be able to help here.  He seems to be the SME.

Reply:
The node process DNS needs to be on localhost and 53 in order to not conflict with dnsmasq.

Reply:
This is what I've been testing, it would avoid having any additional node specific data in the config file.

```
except-interface=lo
bind-interfaces
```

see https://github.com/openshift/openshift-ansible/pull/5707

Reply:
Are you wanting the DNS settings removed?  This PR referenced is yet to be merged.  I'm fine with making those changes and testing but I'm unsure with what the plan is with those changes.